app/controllers/application_controller.rb

```diff
@@ -153,6 +162,6 @@ def append_info_to_payload(payload)
   end
 
   def redesign?
-    @affiliate.use_redesigned_results_page
+    @affiliate.use_redesigned_results_page or permitted_params[:redesign] == 'true'
```

Comment:
small improvement but what do you think about using `permitted_params[:redesign].present?` here as well, that way we can set the variable to anything to enable it.

Reply:
We have some tests using the hard-coded `true`. This PR would need more changes to use `#present?`.